libs/gi/db/src/Database/DataManagers/OptConfigDataManager.ts

```diff
@@ -161,8 +153,7 @@ export class OptConfigDataManager extends DataManager<
 
     excludedLocations = validateArr(
       excludedLocations,
-      allLocationCharacterKeys.filter((k) => k !== key),
-      [] // Remove self from list
+      allLocationCharacterKeys
```

Comment:
Please revert to previous implementation.

Reply:
I changed this because it was changed in #2522. I reverted the change as you asked.